src/signature.rs

```diff
@@ -306,7 +306,7 @@ pub fn verify_messages(
 mod tests {
     use super::*;
 
-    use base64::STANDARD;
+    use base64::prelude::*;
```

Comment:
I personally prefer not having glob imports. Here's only for tests, which could be OK, but importing the concrete things should be easy enough :)

Reply:
Yeah, I can do that.